packages/axios-extension/src/abap/types/adtTypes.ts

```diff
@@ -0,0 +1,30 @@
+export interface AdtSchemaData {
```

Comment:
Minor thing: we used `types.ts` as name in other places when defining types and interfaces